android/app/src/main/res/layout/launch_screen.xml

```diff
@@ -1,6 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:orientation="vertical" android:layout_width="match_parent"
-    android:layout_height="match_parent">
-    <ImageView android:layout_width="match_parent" android:layout_height="match_parent" android:src="@drawable/splash_screen" android:scaleType="centerCrop" />
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/helio_primary">
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `RelativeLayout` has an attribute `android:orientation="vertical"` which is not applicable to this layout. This attribute is typically used with LinearLayout. Since RelativeLayout does not use this attribute, it can be safely removed.
```diff
-    android:orientation="vertical"
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->